### PR TITLE
HW1: adding proposal for SportsCardTool, adding numpy project TLG2141

### PR DESCRIPTION
**Pull request by TravisGibbs**

### CODE REVIEW

**Comment by timkpaine** [comment]
lgtm

File: projects/python/numpy.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Numpy
+
+[![](https://img.shields.io/badge/Numpy-docs-green)](https://numpy.org)
+[![](https://img.shields.io/badge/Numpy-repo-blue)](https://github.com/numpy/numpy)
+
+NumPy is a fundamental package for scientific computing in Python. It provides the building blocks of linear algebra, statistics, and other numeric manipulation. In doing so it has become fundamental tool in both academic and industry settings.
+
+## Issues
+
+- [BUG: Inconsistent complex-->float casting failures](https://github.com/numpy/numpy/issues/23062)
```
Comment (timkpaine) [inline projects/python/numpy.md:11]: pick an issue for your project, don't 1-1 use my example

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: projects/python/numpy.md (hunk 1 above)
Comment (TravisGibbs) [reply projects/python/numpy.md:11]: Sorry! I thought we could use NumPy still if the issue was different.

File: tlg2141.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+UNI: tlg2141
+
+Github: [link](https://github.com/TravisGibbs)
+
+
+GitHub README: [link](https://github.com/TravisGibbs/TravisGibbs/blob/main/README.md)
+
+
+## Project Proposal
+I am interested in building an API for python that provides quick access for collectable sports card data. Simmilar tools exsist
```
Comment (timkpaine) [inline tlg2141.md:14]: link to project md

File: tlg2141.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+3. Create api tooling allowing for tracking of PC (personal collection) for each user
+
+4. Establish a pipeline for pricing using external sources that should already exsist
+
+
+A more well worked example could be something like this Pokemon tool [link](https://pokemontcg.io/)
+
+
+## Project Contributions
+I am really interested in contributing to a large python library like Pandas, Numpy, or Pytorch.
```
Comment (timkpaine) [inline tlg2141.md:32]: link to project md

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: tlg2141.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+A more well worked example could be something like this Pokemon tool [link](https://pokemontcg.io/)
+
+- [sportscards](./projects/python/sportscards.md)
+
+
+## Project Contributions
+I am really interested in contributing to a large python library like Pandas, Numpy, or Pytorch.
+
+- [pandas](./projects/python/panas.md)
```
Comment (timkpaine) [inline tlg2141.md:36]: typo

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**